src/config/Config.zig

```diff
@@ -1381,6 +1381,22 @@ keybind: Keybinds = .{},
 ///   * `end` - Insert the new tab at the end of the tab list.
 @"window-new-tab-position": WindowNewTabPosition = .current,
 
+/// If `true`, show a numeric icon on the tabs that indicate the position of the
+/// tab. Tabs are numbered starting at 1. On GTK the icons appear as a number
+/// inside a circle on the left side of the tab. On macOS the icons appear as
+/// `⌘N` on the right side of the tab.
+///
+/// By default, Ghostty has keybinds for directly switching to tabs 1-8, using
+/// `Alt+N` (`Cmd+N` on macOS). `Alt+9` (`Cmd+9` on macOS) is configured by
+/// default to go to the last tab. These keybinds always work and are not
+/// affected by the state of the tabs icon.
+///
+/// The default is `false` (no icons will be shown) except on macOS where the
+/// default is `true`.
+///
+/// Disabling the icons does not work on macOS.
+@"window-tab-icons": bool = if (builtin.os.tag.isDarwin()) true else false,
```

Comment:
If we really want this feature, I kind of feel we should make it a `gtk-`-prefixed feature. It doesn't make sense to me on macOS, I don't know any application that has anything like this.

Reply:
Switched back to `gtk-` prefix.